tests/test_calculator.hh

```diff
@@ -715,72 +748,114 @@ namespace rascal {
         result_idx += n_entries_per_key;
       }
       for (auto neigh : center) {
+        if (static_cast<size_t>(neigh.get_atom_tag()) >=
```

Comment:
same with is_ghost

Reply:
yeah yeah, it's in progress (see the TODO earlier)